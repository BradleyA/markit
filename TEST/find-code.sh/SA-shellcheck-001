/home/uadmin/github/BradleyA/markit/hooks/EXAMPLES/SA-shellcheck-001