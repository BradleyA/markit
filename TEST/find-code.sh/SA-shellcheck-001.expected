
In SA-shellcheck-001 line 227:
    ssh -t "${USER}"@"${NODE}" "echo -e '${BOLD}${YELLOW}\c' ; ${REMOTE_COMMAND} ; echo -e '${NORMAL}\c'"
                                         ^-- SC2029: Note that, unescaped, this expands on the client side.

